TempForceSensor/CS/README.md

```diff
@@ -0,0 +1,234 @@
+---
+page_type: sample
+urlFragment: temp-force-sensor
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise
+description: Control a basic temperature and force sensor for Windows 10 IoT Enterprise.
+---
+
+# Temperature and force sensor
+This sample uses SPI communication. 
+
+A temperature/force sensor is connected to an ADC, the ADC is connected to the UP Board through SPI Pins. 
+
+The ADC converts the analog sensor output to a digital value that is then read by the UP Board using SPI. 
+
+The value read from the ADC is displayed on the screen attached to the UP Board.
+
+This is basically a simplified version of Potentiometer sensor sample, which has an LED light as an extra output.
+
+You can also use a Force sensor in this sample. Try to press the Force sensor gentle or hard to see the data output difference.
+
+This sample only has C# version.
+
+## Read before start
+This sample assumes that UP Board has already been set up as below:
+
+- UP Board has been connected to HDMI monitor
+- A SD image card has been plugged to UP Board
+- An Ethernet cable has been plugged to UP Board
+- UP Board has been powered on
+
+## Parts needed
+
+- [1 MCP3002 10-bit ADC](http://www.digikey.com/product-detail/en/MCP3002-I%2FP/MCP3002-I%2FP-ND/319412) or [1 MCP3208 12-bit ADC](http://www.digikey.com/product-search/en?KeyWords=mcp3208%20ci%2Fp&WT.z_header=search_go)
+- [1 TMP36 Temperature sensor](http://www.digikey.com/product-detail/en/TMP36GT9Z/TMP36GT9Z-ND/820404) or [1 FSR 402 Force sensor](http://www.digikey.com/product-detail/en/30-81794/1027-1001-ND/2476468)
+- UP Board
+- 1 breadboard and a couple of wires
+- HDMI Monitor
+
+## Parts Review
+
+* MCP3002 or MCP3208
+
+Below are the pinouts of the MCP3002 and MCP3208 analog-to-digital converters (ADC) used in this sample.
+
+![Electrical Components](../../Resources/MCP3002.png)
+![Electrical Components](../../Resources/MCP3208.png)
+
+* UP Board
+
+ ![UP Board Pinout](../../Resources/UP Board_Pinout.png)

```

Comment:
Make sure this picture is visible

Reply:
File path of the image is updated, It is visible now.